### DISCUSSION TITLE
System specification for DL

### DISCUSSION THREAD

**Discussion by yogi251579** [community/community#144396] [Programming Help]
### Select Topic Area

Question

### Body

Could you please provide detailed insights into the system specifications required for deep learning-based object detection tasks?

**Comment by hrithikhadka** [comment]
Hello,

For deep learning data-intensive heavy tasks, you'll probably want to prioritize the following:
1) NVIDIA RTX 3090 with 24GB VRAM
2) AMD Ryzen 9 or Intel i9
3) 32GB RAM or above
4) 2TB SSD storage

**Reply by yogi251579** [reply]
Thanks a lot bro....

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @yogi251579!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
